src/api/schemas.py

```diff
@@ -148,3 +149,12 @@ def to_orm(self) -> ExternalSiteUser:
             last_name=self.last_name,
             specializations=self.specializations,
         )
+
+    @field_validator("specializations")
+    def specializations_str_validation(cls, value: str):
+        if isinstance(value, str):
```

Comment:
Как то много вложенности - попробуй `not`: 
```suggestion
        if not isinstance(value, str):
            return value
```